input/pagecontent/terminology.md

```diff
@@ -53,7 +52,6 @@ New code systems were created when no existing code systems were deemed fit for
 
 |  Code System | Application |
 |--------------|-------------|
-| Radiotherapy Code System | Radiotherapy modalities, techniques, and body sites |
```

Comment:
@saulakravitz some more un-related changes snuck into this branch -- not sure if this matters or not

Reply:
Yes, they snuck in, but they need to happen.